{"name": "故障告警列表", "status": "passed", "attachments": [{"name": "log", "source": "5c960874-0692-4e12-8c4d-5b6be5b6cc20-attachment.txt", "type": "text/plain"}], "start": 1690529581192, "stop": 1690529581528, "uuid": "f6e745d9-9884-4992-b418-9bdcb31a9983", "historyId": "0282e3e93269cab386dfdc5d22673b0d", "testCaseId": "62fd060fc57685ab01a373c2afd7152e", "fullName": "opsPC.test_alarm#test_alarm_fault_list_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_alarm"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "11532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_alarm"}]}